src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task {
+    protected String ddate;
```

Comment:
Maybe could use a more descriptive name for this? For example dueDate instead of just ddate.